src/components/views/rooms/RoomTooltip.js

```diff
@@ -39,6 +41,13 @@ module.exports = React.createClass({
         this.tooltipContainer = document.createElement("div");
         this.tooltipContainer.className = "mx_RoomTileTooltip_wrapper";
         document.body.appendChild(this.tooltipContainer);
+        window.addEventListener('scroll', this._renderTooltip, true);
+
+        const parent = ReactDOM.findDOMNode(this).parentNode;
+        this.state={
+            parent,
+            parentBox: parent.getBoundingClientRect(),
+        };
```

Comment:
These don't need to be part of the state, in fact this will cause un unnecessary render after the first one because the state's changing.